app/src/main/java/tech/ula/utils/AndroidUtility.kt

```diff
@@ -85,17 +82,29 @@ class DefaultPreferences(private val prefs: SharedPreferences) {
 }
 
 class AssetPreferences(private val prefs: SharedPreferences) {
-    private fun String.addTimestampPrefix(): String {
-        return "timestamp-" + this
+
+    private val versionString = "version"
```

Comment:
It appears this string is a constant so should we change the variable name to all caps to signify that it won't change?

I was reading about it in this stackoverflow post: 
https://stackoverflow.com/questions/44038721/constants-in-kotlin-whats-a-recommended-way-to-create-them

It may help readability, particularly when using them in the string templates (for example line 100)

Reply:
It looks like that's not actually the case for private variables. I think that convention is meant for global and static constants.
https://discuss.kotlinlang.org/t/naming-convention-for-private-constants/8871

Renaming it according to your suggestions makes the linter complain.

Reply:
![image](https://user-images.githubusercontent.com/10427470/55028678-892ba000-4fc5-11e9-856e-5dd5e8b9e560.png)